### ISSUE TITLE
Allow to provide a default value to getValue function

### ISSUE THREAD

**Issue by ppalomera** [mrmilu/drupal_site_config#4]
Instead of returning an empty array, we should have the option to define a default value to return if our config is empty. Like is done in drupal state and drupal settings:

```php
public function getValue($siteKey, $field, $defaultValue = []) {}
```

**Comment by jherencia** [comment]
I do not see why `[]` is a valid $defaultValue, it should be `NULL` instead, right?

**Comment by ppalomera** [comment]
> I do not see why `[]` is a valid $defaultValue, it should be `NULL` instead, right?

Yes, I assumed `[]` as default value to be compliant with current solution, but I have changed it to `NULL`
